datajoint/erd.py

```diff
@@ -208,7 +208,11 @@ def _make_graph(self):
             graph = nx.DiGraph(self).subgraph(nodes)
             nx.set_node_attributes(graph, 'node_type', {n: _get_tier(n) for n in graph})
             # relabel nodes to class names
-            mapping = {node: (lookup_class_name(node, self.context) or node) for node in graph.nodes()}
+            mapping = {node: (node) for node in graph.nodes()}
```

Comment:
looks like this value of `mapping` is computed but not used.

Reply:
ah yes. looks like my temp code bled through. Yay Code Review!